README.md

```diff
@@ -45,7 +52,9 @@ var add = (x, y) => {
 }
 ```
 
--- YOUR ANSWER HERE --
+ES6
+not anonymous
```

Comment:
It is anonymous. This is also a **function expression**. But it uses a fat arrow function. Fat arrow functions are anonymous by definition because they have no name